src/router.ts

```diff
@@ -24,8 +24,11 @@ export function createRouter(withRouter: (router: Router) => Router = identity):
     .use('/v1', v1Router.routes(), v1Router.allowedMethods())
 
   // tslint:disable-next-line:no-expression-statement
-  readdirSync(process.cwd() + '/public').forEach(fileName =>
-    router.get('/' + fileName, ctx => send(ctx, `/public/${fileName}`)))
+  readdirSync(process.cwd() + '/public').forEach(fileName => {
+    const file = parse(fileName)
+    const route = file.ext === '.html' ? file.name : fileName
+    return router.get('/' + route, ctx => send(ctx, `/public/${fileName}`))
```

Comment:
Are there any benefits for creating a route for each static file? Why not using `koa-static`?

Reply:
Thanks for bringing that up. We want `GET /welcome` to be the welcome page, not `GET /welcome.html`. Took a look at the source code and it looks like they're doing pretty much what we're doing, but not sure how to configure it to strip the `.html` for koa files. This is working nicely, so let's leave it alone.